api/v1/views/__init__.py

```diff
@@ -4,3 +4,5 @@
 def generate_response(model_result):
     if 'Invalid Id' in model_result:
         return make_response(jsonify({"status": 404, "error": "Invalid Id Not Found"}), 404)
+    # Fix For Delete Bug If Item Doesnt Exist- FIxes Deleting Twice
```

Comment:
Shouldn't both status coded be 404?

Reply:
also fix merge conflicts

Reply:
I had thought of it as a bad request but i think the 404 makes more sense,thanks for the rectification,ill fix the conflicts